redaxo/src/addons/be_style/boot.php

```diff
@@ -36,10 +36,22 @@
 
             // Compiled file to copy in frontend assets dir
             rex_file::copy($this->getPath('assets/css/styles.css'), $this->getAssetsPath('css/styles.css'));
+
+
+            $compiler = new rex_scss_compiler();
```

Comment:
Dieses „doppelte“ compile wurde - soweit ich es verstanden hatte - bewusst von @bloep ausgebaut.
Warum brauchen wir es hier wieder?

Reply:
Das kompilieren geht über den ExtensionPoint `BE_STYLE_SCSS_COMPILE`. Den habe ich hier aber nicht in meinem PR, so dass diese Stelle späte noch angepasst werden muss.

Insg. wird das be_style AddOn dann drei Dateien kompilieren.

| SCSS | CSS Pfad | |
|---|---|---|
| `be_style/scss/master.scss` | `/assets/addons/be_style/css/styles.css` | Basis CSS, BS3 Icons  |
| `be_style/scss/master_minibar.scss` | `/assets/addons/be_style/css/minibar.css` | Minibar für FE + BE |
| `be_style/plugins/redaxo/scss/master.scss` | `/assets/addons/be_style/plugins/redaxo/css/styles.css` | Aktuelles Theme |

Reply:
> Den habe ich hier aber nicht in meinem PR, so dass diese Stelle späte noch angepasst werden muss.

rebase?